bindings/nodejs/examples/8-migration.js

```diff
@@ -71,7 +71,7 @@ async function run() {
     // create bundles with the inputs
     for (inputs of input_batches) {
       try {
-        const bundle = await manager.createMigrationBundle(seed, migrationData.inputs.map(input => input.index), {
+        const bundle = await manager.createMigrationBundle(seed, inputs.map(input => input.index), {
```

Comment:
It should be `inputs.inputs` as `inputs` is an object which a an `inputs` array embedded inside. Perhaps, it is clearer if we rename `inputs` to `batch` so it will become `batch for input_batches` and then we can access it `batch.inputs`